prog/weavedns/main.go

```diff
@@ -95,14 +96,18 @@ func main() {
 	if err != nil {
 		Error.Fatal("[main] Unable to initialize the Zone database", err)
 	}
-	err = zone.Start()
-	if err != nil {
+	if err := zone.Start(); err != nil {
 		Error.Fatal("[main] Unable to start the Zone database", err)
 	}
 	defer zone.Stop()
 
+	dockerCli, err = docker.NewClient(apiPath)
+	if err != nil {
+		Error.Fatal("[main] Unable to start docker client", err)
+	}
```

Comment:
well spotted. looks like `Start()` isn't called anywhere. I suggest we fold the `Start()` behaviour into `NewClient()`.